### PR TITLE
add example for excluding content using the --pattern cmd line arg

### PR DESCRIPTION
**Pull request by mherkazandjian**
This pull request addresses #5176 
it is just a simple contribution that i find useful myself.

### CODE REVIEW

**Comment by mherkazandjian** [comment]
i checked how it renders in sphinx locally, looks good.
image

File: docs/usage/list.rst [hunk 1]
```diff
@@ -19,6 +19,12 @@ Examples
     -rwxr-xr-x root   root       2140 Fri, 2015-03-27 20:24:22 bin/bzdiff
     ...
 
+    $ borg list /path/to/repo::root-2016-02-15 --pattern "- bin/bz*"
+    drwxr-xr-x root   root          0 Mon, 2016-02-15 17:44:27 .
+    drwxrwxr-x root   root          0 Mon, 2016-02-15 19:04:49 bin
+    -rwxr-xr-x root   root    1029624 Thu, 2014-11-13 00:08:51 bin/bash
+    ...
+
```
Comment (ThomasWaldmann) [inline docs/usage/list.rst:27]: well, that leaves the user to guess about what "..." will show (and what not).

how about using "- bin/ba*" as pattern so this is more clear?

Comment (mherkazandjian) [reply docs/usage/list.rst]: done ec0c3a8a
tnx for the comment.

image

**Comment by ThomasWaldmann** [comment]
thanks! btw, the 1.1-maint branch has the most used docs for borg 1.1.x, would you like to backport this to 1.1-maint?

**Comment by mherkazandjian** [comment]
> thanks! btw, the 1.1-maint branch has the most used docs for borg 1.1.x, would you like to backport this to 1.1-maint?

yes sure, its a quite easy fix, done ( #5458 )

**Pull request merged**